### PR TITLE
Hero Section Revamp

### PR DESCRIPTION
**Pull request by Ellahinator**
- https://github.com/trypear/pear-landing-page/issues/59

Figma:
image

Before: 
image

Implementation: 
image

Notes:
- ~~I had trouble adding the logo to the left of "PearAI:" and would like some help.~~
  - @somwrks added the logo to the hero https://github.com/Ellahinator/pear-landing-page/pull/6

- Also am unsure what the `hover` behavior should be for the `More details` button.

### CODE REVIEW

File: components/hero.tsx
Comment (MohammedADev) [inline components/hero.tsx]: It's a good practice to provide alternative text (alt attribute) for the <PearHeroLogo> component just in case the image fails to load or for screen reader users

File: components/hero.tsx
Comment (MohammedADev) [inline components/hero.tsx]: remove unnecessary comment

File: components/hero.tsx
Comment (MohammedADev) [inline components/hero.tsx]: replace `you'll` with `you&apos;ll` otherwise Eslint will throw error

**Review by MohammedADev — CHANGES_REQUESTED** [review]
very minor suggestions to improve code quality

File: components/hero.tsx
Comment (MohammedADev) [reply components/hero.tsx]: that would be for both PearHeroLogo components, and the same alt text can be used for both

Comment (Ellahinator) [reply components/hero.tsx]: Done!

File: components/hero.tsx
Comment (Ellahinator) [reply components/hero.tsx]: Done!

File: components/hero.tsx
Comment (Ellahinator) [reply components/hero.tsx]: Done!

**Review by MohammedADev — APPROVED** [review]
Shouldn't merge before add hover effect

**Comment by Fryingpannn** [comment]
## Great PR! Just 2 things to change

1. Fix mobile responsiveness :) the "logo+pearai" should be its own line
image

2. For hover effect, accentuates the color more sharply/darker + slight shadow should do the trick!
image
image

Good to go after these!

**Comment by Ellahinator** [comment]
1. Fixed responsiveness for all sizes
2.  Added subtle hover effect on "More Details" button
    - darker border and darker text + shadow. also added shadow on "Join Waitlist" button
 
    image

    image

**Review by Fryingpannn — APPROVED** [review]

**Pull request merged**
